aws-resources/src/test/java/io/opentelemetry/contrib/aws/resource/ResourceComponentProviderTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.aws.resource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.sdk.autoconfigure.internal.SpiHelper;
+import io.opentelemetry.sdk.autoconfigure.spi.internal.ComponentProvider;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class ResourceComponentProviderTest {
+
+  @Test
+  @SuppressWarnings("rawtypes")
+  void providerIsLoaded() {
+    List<ComponentProvider> providers =
+        SpiHelper.create(ResourceComponentProviderTest.class.getClassLoader())
```

Comment:
Can do this with `ComponentLoader` from the new `opentelemetry-common` package as of https://github.com/open-telemetry/opentelemetry-java/pull/7446, which is first available in opentelemetry-java 1.52.0 release.